specification/metadata/schema/datatype.md

```diff
@@ -0,0 +1,17 @@
+# Data Type
+
+The data type of the column provided in the FOCUS dataset.
```

Comment:
I think we had a discussion that this needs to be the specific datatype used by the provider (not high level data type like what FOCUS defines - e.g. String/Numeric etc.)

Reply:
I would like to see this with a set of allowed values.

Reply:
Open to suggestions here

Reply:
Im struggling to understand how we can have this as specific and an allowable list of values.

Reply:
Maybe the data type reference of the provider (BigQuery, MS SQL, ...) could be specified as a column in metadata: allowed values for column data types would be data types in this reference.